Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2208,6 +2280,18 @@ def parse_dict_ticket_fields(client: Client, ticket: dict) -> dict:
     return ticket
 
 
+def get_timezone_offset(full_response):
+    """
+    Receives the full response of a ticket query from SNOW and computes the timezone offset between the timezone of the
+    instance and UTC.
+    """
+    local_time = datetime.strptime(full_response.get('result', {}).get('sys_created_on', {}).get('display_value', ''),
+                                   DATE_FORMAT)
+    utc_time = datetime.strptime(full_response.get('result', {}).get('sys_created_on', {}).get('value', ''), DATE_FORMAT)
+    offset = utc_time - local_time
+    return offset
```

Comment:
you should try/catch each parsing here separately and add more information to the exception that will explain what we failed to parse.